src/macaron/config/defaults.ini

```diff
@@ -318,6 +318,60 @@ deploy_arg =
 github_actions =
     docker/build-push-action
 
+# This is the spec for trusted NPM build tool usages.
+[builder.npm]
+entry_conf =
+    .npmrc
+    .yarnrc
+build_configs =
+    package.json
+package_lock =
+    package-lock.json
+builder =
+    npm
+    yarn
+build_arg =
+    build
+deploy_arg =
+    publish
+[builder.npm.ci.deploy]
+github_actions =
+    JS-DevTools/npm-publish
+
+# This is the spec for trusted Yarn build tool usages.
+[builder.yarn]
+entry_conf =
+    .yarnrc
+build_configs =
+    package.json
+package_lock =
+    package-lock.json
+builder =
+    yarn
+build_arg =
+    build
+deploy_arg =
+    publish
+
+# This is the spec for trusted Go build tool usages.
+[builder.go]
+entry_conf =
+build_configs =
```

Comment:
Have you looked into `go.sum` where the direct and indirect dependencies are listed? (see [here](https://go.dev/ref/mod#go-sum-files))

Reply:
Didn't consider it, can add it